tests/contrib/validation/cerberus/test_validator_cerberus.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+import pytest
+from spidermon.contrib.validation import CerberusValidator
+
+
+@pytest.mark.parametrize(
+    "data,schema",
+    [
+        pytest.param(
+            {"name": "foo", "number": 5},
+            {"name": {"typestring"}, "number": {"type": "integer"}},
+            id="Schema Invalid",
+        ),
+        pytest.param(
+            ["This is wrong"],
+            {"name": {"type": "string"}, "number": {"type": "integer"}},
+            id="Data invalid, not mapping type",
+        ),
+        pytest.param(
+            ["foo"],
+            {"name": {"boo"}, "number": {"type": "integer"}},
+            id="Both schema, and data invalid",
+        ),
+    ],
+)
+def test_function(data, schema):
```

Comment:
All parameters are invalid schemas? So your test name should be more informative like `test_raise_value_error_with_invalid_schemas` or something like that.  `test_function` is not a good test name.

Reply:
Also, where are the tests with valid schemas? You need to verify if the happy-path is working too.

Reply:
That's on me, I think between deleting and stuff I might have misplaced the name you actually suggested in the first place. More descriptive test names coming right up. 

About Valid schemas, we can't actually do that without raising an assertion error, in this case, can we. I can introduce a property of `valid=True` but then we have to also have expected errors, as they go hand in hand with the output of validator if that works? I personally also like to verify the happy path but would like some insight on the best way of implementing it.

Reply:
For a happy path, it would be a new test function.
`test_invalid_schema` and all it's test cases
`test_valid_schema` and all it's  test cases (if it applies for more than one test)